packages/core/src/types-hoist/index.ts

```diff
@@ -19,6 +19,7 @@ export type {
   TraceContext,
   CloudResourceContext,
   MissingInstrumentationContext,
+  FeatureFlagContext,
```

Comment:
🤔 Is this used anywhere in the SDK?

Reply:
If not directly, then in the Contexts type.

Reply:
I don't see it used anywhere